packages/jade-compiler/tests/tests.js

```diff
@@ -30,3 +30,49 @@ Tinytest.add("JadeCompiler - compile templates", function(test) {
   test.equal(JadeCompiler.compile(template),
   "(function() {\n  return HTML.P(\"hello world\");\n})");
 });
+
+var template3 = wrapInTemplate("hello",
+  ["if helper arg1 arg2",
```

Comment:
I think we can use ES6 template string here, just add the meteor `ecmascript` package and replace `"` by `\``